docs/internals/sorting.md

```diff
@@ -0,0 +1,93 @@
+# Sorting
+
+Quickwit can sort results based on fastfield values or score. This document discuss where and ho
+ it happens.
+It also tries to describe optimizations that may be enabled (but are not necessarily implemente)
+by this behavior.
+
+# Current state
+(this section shall be removed once proposed changes are accepted)
+
+## Behavior
+
+Sorting is controlled by the `sort_by` query parameter. It accepts a comma separated list of fields
+to use for sorting. Sorting is Ascending by default. The sorting order can be reversed by prefixing
+a field name with a hyphen `-`.
+The special value `_score` means sorting by score, with a sort that is then Descending by default
+(documents matching better goes first).
+
+In case of equality between two documents, the GlobalDocId, composed of (SplitId, SegmentId, DocId)
+is used as a tie breaker. It is always used in Ascending order *inside a split*. It is used in the
```

Comment:
This behavior is not great. You are offering a spec where the repartition of docs in splits as an effect on search results.

Could we do ascending always?